portfolio/src/main/java/com/google/sps/utils/ServletUtils.java

```diff
@@ -18,4 +18,27 @@ public static String getParameter(HttpServletRequest request, String name, Strin
         }
         return value;
     }
+
+    /**
+     * Gets the integer parameter from html form input.
+     * @param request Http request.
+     * @param parameterName Input id in html element.
+     * @param defaultValue Default integer.
+     * @return Parsed integer or default value if exception occur
+     */
+    public static int getIntParameter(HttpServletRequest request, String parameterName, int defaultValue) {
+        String maxNumOfCommentStr = request.getParameter(parameterName);
+
+        int result = defaultValue;
+
+        try {
+            result = Integer.parseInt(maxNumOfCommentStr);
+        } catch (NumberFormatException e) {
+            System.err.println("Could not convert to int: " + maxNumOfCommentStr);
+
+            return result;
```

Comment:
nit: return defaultValue <- this makes the code more explicit / readable than relying on the defaultvalue set in result.

Reply:
Fixed